src/java.base/share/classes/jdk/internal/util/random/RandomSupport.java

```diff
@@ -1376,7 +1408,7 @@ public static double computeNextGaussian(RandomGenerator rng) {
             for (;; U1 = (rng.nextLong() >>> 1)) {
                 long U2 = (rng.nextLong() >>> 1);
                 // Compute the actual x-coordinate of the randomly chosen point.
-                x = (X[j] * 0x1.0p63) + ((X[j-1] - X[j]) * (double)U1);
+                x = Math.fma(X[j-1] - X[j], (double)U1, X[j] * 0x1.0p63));
```

Comment:
Code is not compilable

Reply:
@Pr0methean As @turbanoff observes, I think there's a closing parentheses too much on L.1411 and one on L.1421.
Which compiler are you using ;-)  ?

Reply:
Should be fixed as of fa340fb47b1169805f21fb71cce0da3fc175d427. I haven't yet found a way for IntelliJ IDEA to use the project as its own JDK, so I've sanity-checked the syntax using the Rainbow Brackets plugin.

Reply:
Update: successfully built `images` and `run-test-tier1` on my AL2 machine.